core/unit_test/TestPolicyConstruction.hpp

```diff
@@ -584,84 +585,84 @@ class TestTeamPolicyConstruction {
 #ifndef KOKKOS_ENABLE_SYCL
     ASSERT_GT(p1.chunk_size(), 0);
 #endif
-    ASSERT_EQ(p1.scratch_size(0), 0);
```

Comment:
This has to be converted to unsigned on both sides, because `scratch_size` returns `int` for CUDA (`size_t` otherwise):
https://github.com/kokkos/kokkos/blob/e4831441fb213bd29485327b02cf1b54149a58a7/core/src/Cuda/Kokkos_Cuda_Parallel.hpp#L255

Reply:
OK.
Side note: we might want to consider fixing the return type elsewhere.